src/xcode/ENA/ENA/Source/Scenes/Events/Checkin/TraceLocationCheckin/TraceLocationCheckinViewController.swift

```diff
@@ -98,15 +95,35 @@ class TraceLocationCheckinViewController: UIViewController {
 			}
 			.store(in: &subscriptions)
 	}
+	
+	private func setupNavigationBar() {
+		let logoImage = UIImage(imageLiteralResourceName: "Corona-Warn-App-Small").withRenderingMode(.alwaysTemplate)
+		let logoImageView = UIImageView(image: logoImage)
+		logoImageView.tintColor = .enaColor(for: .textContrast)
+		
+		navigationController?.navigationBar.tintColor = .white
+		navigationItem.leftBarButtonItem = UIBarButtonItem(customView: logoImageView)
+
+		let closeButton = dismissHandlingCloseBarButton(.contrast)
+		closeButton.accessibilityLabel = AppStrings.AccessibilityLabel.close
+		closeButton.accessibilityIdentifier = AccessibilityIdentifiers.AccessibilityLabel.close
+		navigationItem.rightBarButtonItem = closeButton
+		navigationItem.hidesBackButton = true
+
+		// create a transparent navigation bar
+		let emptyImage = UIImage()
+		navigationController?.navigationBar.setBackgroundImage(emptyImage, for: .default)
+		navigationController?.navigationBar.shadowImage = emptyImage
+		navigationController?.navigationBar.isTranslucent = true
+		navigationController?.view.backgroundColor = .clear
```

Comment:
This is only needed if you use a normal UINavigationController, but if you do so, the DismissHandling protocol might not work correctly.